share/rocm/cmake/ROCMInstallSymlinks.cmake

```diff
@@ -3,6 +3,11 @@
 # ######################################################################################################################
 
 function(rocm_install_symlink_subdir SUBDIR)
+    set(options)
+    set(oneValueArgs COMPONENT NAME_PREFIX)
```

Comment:
It is a prefix that is added to the symlink, it is intended to help avoid name collisions (for example, the math libraries currently generate example executables named like `example_xyz`, where `xyz` is independent of the library and may be reused between libraries. Allowing a prefix to be added resolves this collision, and provides  information at a glance on which library installed a particular example.

Reply:
Why not just set the `OUTPUT_NAME` property to the unique name? You can then add a `POST_BUILD` custom command to symlink(or copy) the short name in the build tree. You could add a function to take care of this.

It can be confusing to have two seperate names. A user script could fail if they were to add /opt/rocm/bin vs /opt/rocm/rocblas/bin to the path.